neuron_morphology/transforms/scale_correction/compute_scale_correction.py

```diff
@@ -0,0 +1,107 @@
+from typing import Dict, Optional, List, Any
+
+import logging
+import copy as cp
+
+import numpy as np
+import pandas as pd
+
+from argschema.argschema_parser import ArgSchemaParser
+from neuron_morphology.transforms.scale_correction._schemas import (
+    InputParameters, OutputParameters)
+from neuron_morphology.morphology import Morphology
+from neuron_morphology.swc_io import morphology_from_swc
+import neuron_morphology.transforms.affine_transform as aff
+
+
+def get_scale_correction(morphology: Morphology,
+                         cell_depth: float,
+                         soma_marker_z: float,
+                         cut_thickness: Optional[float] = 350):
+    """
+        Estimate a scale factor correction from recorded cell depth
```

Comment:
Could you please describe here the problem of scaling the morphology. What is a fixed depth and why we are dividing cell_depth by the fixed depth.